### PR TITLE
Add forking / branch workflow image

### PR DESCRIPTION
**Pull request by taylorfturner**

### CODE REVIEW

**Review by JGSweets — COMMENTED** [review]
in terms of staging, those would be individual new branches from main or dev, right? I think the continued line is a bit confusing.

**Comment by taylorfturner** [comment]
> What's the extra arrow in main?

Shouldn't be there -- didn't update that when I changed something else. good call. updating on next commit

**Comment by taylorfturner** [comment]
> in terms of staging, those would be individual new branches from main or dev, right? I think the continued line is a bit confusing.

yeah, removed that continued line. Effectively its just a branch to literally stage either the feature into dev or dev into main (for a release). So the diagram is drawn in such a way to notate that there should be no commits to those staging branches.  @JGSweets @micdavis @ksneab7 @tyfarnan

**Comment by taylorfturner** [comment]
related to #857

File: docs/source/_static/images/branching_workflow_diagram.png
Comment (tyfarnan) [inline docs/source/_static/images/branching_workflow_diagram.png:1]: pretty!

Comment (taylorfturner) [reply docs/source/_static/images/branching_workflow_diagram.png:1]: ty

**Pull request merged**